### PR TITLE
Prevent use of TLS 1.1

### PR DESCRIPTION
**Pull request by moshekaplan**

### CODE REVIEW

**Comment by B0fH** [comment]
Hi @moshekaplan - 
 Kindly review the comment I've posted above.

Thanks,
 Elazar

**Comment by moshekaplan** [comment]
> Hi @moshekaplan - Kindly review the comment I've posted above.
> 
> Thanks, Elazar

Which comment is this? I don't see any.

**Review by B0fH — CHANGES_REQUESTED** [review]
Hi @moshekaplan -
 Kindly restore the newline and then I can go ahead and merge this.
Thanks,
 Elazar

File: hxtool.py [hunk 1]
```diff
@@ -725,4 +726,4 @@ def hxtool_upgrade():
 				
 else:
 	# Running under gunicorn/mod_wsgi
-	app_init(debug = False)
```
[the remark this answers is not in the document]
Comment (moshekaplan) [reply hxtool.py:728]: I'll try to figure out how. Github's online editor automatically added the newline at the end.

[the remark this answers is not in the document]
Comment (moshekaplan) [reply hxtool.py]: Fixed

**Pull request merged**